src/ReaderScreen/hooks/useFetchShabad.js

```diff
@@ -13,25 +13,34 @@ const useFetchShabad = (shabadID) => {
   const isLarivaarAssist = useSelector((state) => state.isLarivaarAssist);
   const isParagraphMode = useSelector((state) => state.isParagraphMode);
   const isVishraam = useSelector((state) => state.isVishraam);
+  // const prevBaniLengthRef = useRef(baniLength);
```

Comment:
We can remove this commented code if not needed.

Reply:
fixed